src/transformers/tokenization_utils_base.py

```diff
@@ -3275,10 +3275,15 @@ def prepare_seq2seq_batch(
             Otherwise, input_ids, attention_mask will be the only keys.
         """
         warnings.warn(
-            "`prepare_seq2seq_batch` is deprecated and will be removed in version 5 of 🤗 Transformers. Use the "
-            "regular `__call__` method to prepare your inputs and the tokenizer under the `as_target_tokenizer` "
-            "context manager to prepare your targets. See the documentation of your specific tokenizer for more "
-            "details",
+            """
+`prepare_seq2seq_batch` is deprecated and will be removed in version 5 of HuggingFace Transformers. To continue using
+it copy-n-paste the internals of this function. The gist of which is:
```

Comment:
I don't see why we would encourage users to continue using a deprecated function.
```suggestion
`prepare_seq2seq_batch` is deprecated and will be removed in version 5 of HuggingFace Transformers. You should use the following snippet instead:
```

Reply:
How would you suggest to rephrase it, since this is not what I meant. 

I meant to communication that to figure out how to replace the deprecated function is to copy the relevant parts of code from it (since it has been brought up to date with the correct code). i.e. the contents of this deprecated function are the best example of how to code the functionality being replaced.

Unless you are trying to communicate that this wrapper does something incorrect/invalid/redundant/non-idiomatic? In which case would you share what that is?

Reply:
I see, I would leave it like in my suggestion before the code example, and then you could add after the code example something along the lines of "For a more complete example, look at the implementation of `prepare_seq2seq_batch`"?

Reply:
I like your proposal. 

Please have a look at the new version. Let me know if I need to move/remove anything.

Thanks for the suggestion on how to over-come auto-wrapper's wrath, @sgugger

Reply:
The output now looks as:
```
/mnt/nvme1/code/huggingface/transformers-prepare_seq2seq_batch/src/transformers/tokenization_utils_base.py:3285: FutureWarning:
`prepare_seq2seq_batch` is deprecated and will be removed in version 5 of HuggingFace Transformers. Use the regular
`__call__` method to prepare your inputs and the tokenizer under the `as_target_tokenizer context manager to prepare
your targets. Here is a short example:

model_inputs = tokenizer(src_texts, ...)
with tokenizer.as_target_tokenizer():
    labels = tokenizer(tgt_texts, ...)
model_inputs["labels"] = labels["input_ids"]


See the documentation of your specific tokenizer for more details on the specific arguments to the tokenizer of choice.
For a more complete example, look at the implementation of prepare_seq2seq_batch.

  warnings.warn(
```

Reply:
Looking good!